torchmd/run.py

```diff
@@ -150,7 +149,7 @@ def get_args(arguments=None):
 precisionmap = {"single": torch.float, "double": torch.double}
 
 
-def setup(args):
+def setup(args, batch_comp=None):
```

Comment:
batch_comp used to be false by default, now it is None, is this really what you want?

Reply:
it works fine in any case, but I will set to false, makes more sense!